build/bundle.go

```diff
@@ -21,13 +21,18 @@ type Bundle struct {
 	// Release is the release id
 	Release string
 	// Path is the (optional) bundle path; takes precedence over url
-	Path string
+	Path map[string]string
 	// URL is the (optional) bundle URL; takes precdence over github release
-	URL string
-	// CHecksum is the bundle sha256 checksume in hex, when specifying a URL.
-	Checksum string
+	URL map[string]BundleURL
```

Comment:
I'd pluralize these fields.

Reply:
maybe, I kind of prefer singular here; but i don't feel strongly either way.